src/store/authentication/authentication.actions.js

```diff
@@ -0,0 +1,9 @@
+function setToken() {

```

Comment:
```suggestion
export function setToken() {
```
export ngay lúc khai báo luôn để đỡ tránh dài dòng khi export quá nhiều action ấy